src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -318,4 +335,36 @@ private void sendUpdateSingleCaseECC(String authToken, CaseDetails currentCaseDe
         }
     }
 
+    public static void populateConciliationTrack(CaseData caseData) {
+        boolean isConTrackOp = false;
+        boolean isConTrackSt = false;
+        boolean isConTrackSh = false;
+        boolean isConTrackNo = false;
+
```

Comment:
Should use Collections.isNotEmpty to check if the jurisdiction collection contains values